source/MultiGrid/MultiGrid.js

```diff
@@ -713,16 +738,32 @@ export default class MultiGrid extends React.PureComponent {
       fixedColumnCount,
       fixedRowCount,
       scrollLeft,
+      hideTopRightGridScrollbar,
     } = props;
     const {showHorizontalScrollbar} = this.state;
 
     if (!fixedRowCount) {
       return null;
     }
 
-    const additionalColumnCount = showHorizontalScrollbar ? 1 : 0;
+    const additionalColumnCount = showHorizontalScrollbar ? 1 : 0,
+      height = this._getTopGridHeight(props),
+      width = this._getRightGridWidth(props),
+      scrollbarSize = this.state.showHorizontalScrollbar ? this.state.scrollbarSize : 0;
```

Comment:
nit: `showHorizontalScrollbar` is already deconstructed above. `scrollbarSize` can also be put there.